config/routes.rb

```diff
@@ -1,8 +1,12 @@
 Rails.application.routes.draw do
   devise_for :users
   root 'main#home'
+  get '/about', to: "main#about"
+  get '/contact', to: "main#contact"
+  get '/pricing', to: "main#pricing"
```

Comment:
As far as I can see those pages are public pages for the website (no login is required), right?

In this case, I would start splitting our project into 2 different modules: `Website` and `App` (choose other names if you want).

`Website` is a module where everything public goes in. Everyone has access to the resources under this module, no authentication is needed.

`App` is a module where everything private goes in, which is the authenticated area. To access any resource from this module, the user needs to be identified and authorized first.

It would be nice to have some documentation on those modules to describe what I described above.

We would end with something like this:

```
Website::PagesController < ApplicationController

App::MainController < BaseController

BaseController < ApplicationController
```

`BaseController` would be where all those things about the devise gem should go in.

All the pages we create for the app should inherit from `BaseController`, what would bring the authentication features.

Maybe you should do that in other PR to not bloat this one.
What do you think?